### PR TITLE
tide documentation

### PR DESCRIPTION
**Pull request by MHBauer**
- link to tide README from prow README
 - clarify reviewApprovedRequired by copying text from k/community

per some discussion in slack, I think this would have avoided some of my questions.

### CODE REVIEW

File: prow/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ not make any attempt to preserve backwards compatibility.
 * `cmd/clonerefs`, `cmd/initupload`, `cmd/gcsupload`, `cmd/entrypoint`, and
   `cmd/sidecar` are small utilities used in `ProwJobs` created by `plank` as
   `Pods`. See [their README](./pod-utilities.md) for more information
+* [`cmd/tide`](./cmd/tide/README.md) manages the testing and merging of PRs
```
Comment (stevekuznetsov) [inline prow/README.md]:
```md
 * `cmd/tide` manages merging PRs once they pass tests and match other criteria. See [its README](./cmd/tide/README.md) for more information
```

File: prow/cmd/tide/README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ tide:
     missingLabels:          // List of can't have labels
     excludedBranches:       // Ignore branches
     includedBranches:       // Include-only branches
-    reviewApprovedRequired: // Review approved is must
+    reviewApprovedRequired: // Github Review approved is must
```
Comment (stevekuznetsov) [inline prow/cmd/tide/README.md]: Maybe `// GitHub Review approval required` ?

File: prow/cmd/tide/README.md [hunk 3]
```diff
@@ -74,7 +74,10 @@ It can consist of the following dictionary of fields:
 * `missingLabels`: List of labels any given PR must not posses.
 * `excludedBranches`: List of branches that get excluded when querying the `repos`.
 * `includedBranches`: List of branches that get included when querying the `repos`.
-* `reviewApprovedRequired`: If set, each PR in the query must have review approved.
+* `reviewApprovedRequired`: defaults to `false`; If set, each PR in
```
Comment (stevekuznetsov) [inline prow/cmd/tide/README.md]: Put default last here, please

**Comment by MHBauer** [comment]
@stevekuznetsov thanks for review. I liked your suggestions.

**Comment by stevekuznetsov** [comment]
Cool!

/lgtm

**Pull request merged**